apps/web/types/types.tsx

```diff
@@ -27,3 +27,17 @@ export type SortOption = {
 	href: string;
 	current: boolean;
 };
+
+export type Opinion = {
+	author: string;
+	avatar: string;
+	rating: number;
+	comment: string;
+};
+
+export type Comment = {
+	author: string;
+	avatar: string;
+	comment: string;
+	date: number;
```

Comment:
Is it UTC timestamp? Are you thinking about specific users from specific timezone? (target)